cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java

```diff
@@ -157,6 +157,7 @@ protected static TestManager getTestManager() {
    * @param applicationClz The application class
    * @return An {@link co.cask.cdap.test.ApplicationManager} to manage the deployed application.
    */
+  @Deprecated
```

Comment:
For the methods in this class marked deprecated, add a link to the method that should be used instead.